src/vsr/replica.zig

```diff
@@ -8377,11 +8397,10 @@ pub fn ReplicaType(
             assert(self.do_view_change_quorum);
             assert(!self.pipeline_repairing);
             assert(self.primary_repair_pipeline() == .done);
+            assert(self.primary_journal_repaired());
 
             assert(self.commit_min == self.commit_max);
             assert(self.commit_max <= self.op);
-            assert(self.journal.dirty.count == 0);
-            assert(self.journal.faulty.count == 0);
             assert(self.valid_hash_chain_between(self.op_repair_min(), self.op));
```

Comment:
Can drop this assert as that is a part of primary_journal_repaired now?

Reply:
Ah, good catch! Removed it as a part of https://github.com/tigerbeetle/tigerbeetle/pull/2414/commits/9da6c0396c43b037c37d7f3067d64341734ac5de. 😄